### ISSUE TITLE
[Bug] Pytorch Mobilenetv3 conversion error.

### ISSUE THREAD

**Issue by pig-pig-yang** [apache/tvm#12469]
I tried to convert Pytorch MobilenetV3 to Relay IR. But the converted Relay IR is wrong. I tried to convert to ONNX first and then convert to Relay, results are still wrong. 

Original structure:
![image](https://user-images.githubusercontent.com/65580829/185101766-2d7eaee7-c3e8-4ee3-8ada-f3e5121ce8ff.png)

Converted Relay IR structure:
![image](https://user-images.githubusercontent.com/65580829/185101669-40a60c11-414c-47cd-9062-d913053300a1.png)

As shown above, the convolution weights are connected to Mul and the branch is connected to conv2D after conversion for every Mul->Conv Node.

The codes are shown below:

```
import tvm
from tvm import te
import tvm.relay as relay
from tvm.contrib.download import download_testdata
import numpy as np
import os

from tvm.relay.quantize.quantize import QConfig

import torch
import torchvision
from torchvision import transforms

from PIL import Image
from matplotlib import pyplot as plt

import netron
import onnx

model_name = "mobilenet_v3"
model = torchvision.models.mobilenet_v3_large(pretrained=True)
model = model.eval()

x = torch.randn(1, 3, 224, 224)
saved_path = "./mobilenet_v3.onnx"
torch.onnx.export(model, x, saved_path)

onnx_model = onnx.load(saved_path)

input_name = "input.1"
shape_list = {input_name: (1, 3, 224, 224)}

mod, params = relay.frontend.from_onnx(onnx_model, shape_list)

print(mod)
```
Since I am not familiar with the frontend,  I don't know what causes the problem.

**Comment by ganler** [comment]
Hi @pig-pig-yang would you like to first verify if the ONNX model itself is correct through [onnx.checker](https://github.com/onnx/onnx/blob/main/docs/PythonAPIOverview.md#checking-an-onnx-model).? It'd be also great if you can paste some error message or just screen shot.

**Comment by pig-pig-yang** [comment]
> Hi @pig-pig-yang would you like to first verify if the ONNX model itself is correct through [onnx.checker](https://github.com/onnx/onnx/blob/main/docs/PythonAPIOverview.md#checking-an-onnx-model).? It'd be also great if you can paste some error message or just screen shot.

Hi, Thanks for the reply!
I have used onnx.checker to check the model is correct. The conversion process using from_pytorch or from_onnx did not report any error, but when I checked the structure of the converted Relay model I found the above structure error.

**Comment by masahi** [comment]
I just checked the output from TVM (using `from_pytorch`) and pytorch. They are identical.

**Comment by masahi** [comment]
In the Relay text dump, I'm seeing this
```
  %57 = clip(%56, a_min=0f, a_max=6f) /* ty=Tensor[(1, 72, 1, 1), float32] */;
  %58 = divide(%57, 6f /* ty=float32 */) /* ty=Tensor[(1, 72, 1, 1), float32] */;
  %59 = multiply(%58, %49) /* ty=Tensor[(1, 72, 28, 28), float32] */;
  %60 = nn.conv2d(%59, %features.4.block.3.0.weight, padding=[0, 0, 0, 0], channels=40, kernel_size=[1, 1]) /* ty=Tensor[(1, 40, 28, 28), float32] */;
```

So it looks all right. @pig-pig-yang Please check your visualization. There is no conversion error.

**Comment by pig-pig-yang** [comment]
> In the Relay text dump, I'm seeing this
> 
> ```
>   %57 = clip(%56, a_min=0f, a_max=6f) /* ty=Tensor[(1, 72, 1, 1), float32] */;
>   %58 = divide(%57, 6f /* ty=float32 */) /* ty=Tensor[(1, 72, 1, 1), float32] */;
>   %59 = multiply(%58, %49) /* ty=Tensor[(1, 72, 28, 28), float32] */;
>   %60 = nn.conv2d(%59, %features.4.block.3.0.weight, padding=[0, 0, 0, 0], channels=40, kernel_size=[1, 1]) /* ty=Tensor[(1, 40, 28, 28), float32] */;
> ```
> 
> So it looks all right. @pig-pig-yang Please check your visualization. There is no conversion error.

Thanks for the reply. Sorry I pasted the wrong codes. I rechecked the codes and the error did not occur during the front-end conversion. The error occurred after `tvm.relay.quantize.prerequisite_optimize`. The codes are:
```
import tvm
from tvm import te
import tvm.relay as relay
from tvm.contrib.download import download_testdata
import numpy as np
import os

import torch
import torchvision
from torchvision import transforms

import netron
import onnx

from tvm.relay.quantize import prerequisite_optimize

##################################################
# Preparing relay networks.
# ------------------------------------------------

model_name = "mobilenet_v3"
model = torchvision.models.mobilenet_v3_large(pretrained=True)
model = model.eval()

input_shape = [1, 3, 224, 224]
input_data = torch.randn(input_shape)
scripted_model = torch.jit.trace(model, input_data).eval()

input_name = "input0"
shape_list = [(input_name, (1, 3, 224, 224))]

mod, params = relay.frontend.from_pytorch(scripted_model, shape_list)

with tvm.transform.PassContext(opt_level=3):
	mod = prerequisite_optimize(mod, params)

print(mod)
```

The `prerequisite_optimize` is consisted of several commonly used pass:
```
def prerequisite_optimize(mod, params=None):
    """Prerequisite optimization passes for quantization. Perform
    "SimplifyInference", "FoldScaleAxis", "FoldConstant", and
    "CanonicalizeOps" optimization before quantization."""
    optimize = tvm.transform.Sequential(
        [
            _transform.SimplifyInference(),
            _transform.FoldConstant(),
            _transform.FoldScaleAxis(),
            _transform.CanonicalizeOps(),
            _transform.FoldConstant(),
        ]
    )

    if params:
        mod["main"] = _bind_params(mod["main"], params)

    mod = optimize(mod)
    return mod
```
Then I gets the wrong Relay structure like this:
  %66 = multiply(meta[relay.Constant][32] /* ty=Tensor[(40, 120, 1, 1), float32] */, %65) /* ty=Tensor[(40, 120, 1, 1), float32] */;
  %67 = nn.conv2d(%54, %66, padding=[0, 0, 0, 0], channels=40, kernel_size=[1, 1]) /* ty=Tensor[(1, 40, 28, 28), float32] */;
  %68 = add(%67, meta[relay.Constant][37] /* ty=Tensor[(40, 1, 1), float32] */) /* ty=Tensor[(1, 40, 28, 28), float32] */;

I haven't confirmed which pass causes this error, I will confirm later. Thanks!
